katas/chess.py

```diff
@@ -1,19 +1,20 @@
-"""Finds the total number of available paths between two spaces on a chessboard while only moving down and right."""
+"""Finds the total number of available paths between two spaces on a chessboard."""
 
 
-def travel_chessboard(s):
-    dx = 1 + int(s[6]) - int(s[1])
-    dy = 1 + int(s[8]) - int(s[3])
+def travel_chessboard(inp):
+    """player can only move down and right."""
+    d_x = 1 + int(inp[6]) - int(inp[1])
+    d_y = 1 + int(inp[8]) - int(inp[3])
     grid = []
-    for col in range(0, dx):
+    for col in range(0, d_x):
```

Comment:
Here's a much more concise way to construct this 2D grid of lists:
`[[1] * d_x for row in d_y]`